### PR TITLE
I fix the UX bug from the Booking Page, and added a StudentBookingDialog

### PR DESCRIPTION
**Pull request by Aljayz**
Fixed some bugs and I added a Dialog intended for booking

### CODE REVIEW

File: src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -93,6 +93,8 @@
       <ConfirmLockDialog />
       <BookAppointmentDialog />
       <RejectApplicationDialog />
+      <StudentBookingDialog />
+      >
```
Comment (29-29) [inline src/layouts/MainLayout.vue]: typo here?

File: src/dialogs/booking/StudentBookingDialog.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      `${typeof row.author == 'object' ? row.author.fullName : ''}`,
+  },
+] as QTableColumn[];
+
+function bookAppointment() {
+  TheDialogs.emit({
+    type: 'bookAppointmentDialog',
+    arg: {
+      success() {
+        console.log();
```
Comment (29-29) [inline src/dialogs/booking/StudentBookingDialog.vue:65]: empty console log?

File: src/layouts/MainLayout.vue (hunk 1 above)
Comment (Aljayz) [reply src/layouts/MainLayout.vue]: Ohh sorry, let me fix that.

**Review by Aljayz — COMMENTED** [review]
I remove the typo, and leave the console log empty, add lang ako later

**Review by fencim — APPROVED** [review]

**Pull request merged**